### DISCUSSION TITLE
170-875 Semi-custom cabinetry?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#75] [General]
Apparently the memory could be separated from the main cabinet...parallel and separate, vs perpendicular and attached as in 60458920's (865/875 HW Ref) Figure 1-2 (photo of 875).  Sorta like an ecm cabinet..  heh..  straight, essentially unobstructed aisles, I'll bet that's why.

Any info on the MAPs? (1-3)???  I'll bet they bought one, or fermilab...  Intel HPF has some really cool features, gfortran compatible for the most part at source level.  DtCyber needs some fortran 😉!  (Nine out of ten people into this, at least under bsd/*ix, probably have gfortran anyway, and Intel's HPF is free now, at least for windows).

![8311750X](https://github.com/kej715/DtCyber/assets/136194499/7ec109f7-23c9-405d-be8b-562799f312ef)

http://cds.cern.ch/record/970275

**Comment by nickglazzard1959** [comment]
This is a very nice photograph (once of a set I use myself as a desktop
background) but I don't think there is a Cyber 865 or 875 in it. I think
this is the CERN computer center circa 1978 and shows their dual CDC 6500,
on the right. A CDC 3200 (? some 3000 series machine(s), not sure what
exactly) in the foreground. CDC 6500 consoles in the center. IBM 370/168 (I
think) in the background. Network and data logging systems (HP and Norsk
Data?) in the left midground. Many of the CDC cabinets are peripheral
controllers, mag tape controllers, mass storage controllers, etc.  Not
visible in this shot is a CDC 7600 which they also had at this time.

This same room is used today for LHC data processing, but its contents are
very different (much more uniform and boring, IMHO).

[image: CernCDSLHC.jpg]
[https://drive.google.com/file/d/18QakNfTi9jVY2rL6xVRkQV13kJfu6qm4/view?usp=sharing](url)

**Reply by KC5VDJ** [reply]
The cern site states it is an 875, and the cabinet config is like a 865 in the mid-right in the photo, but with access panels like the 875 (see the HW Ref, i have a bookmarked version here at project https://github.com/KC5VDJ/CDC-Manuals that you can grab (more coming)).

Interesting, a good computer room never dies.

**Comment by kej715** [comment]
Figures 1-3, 1-4, and 1-5 of the 865/875 HRM show that the memory could
-not- be separated from the main cabinet, and that makes sense since
creating any significant physical distance between the CPU's and memory
would significantly increase the amount of time needed for signals to
travel between them, thus crippling performance (or preventing the system
from running at all). The large machine on the right of the photo matches
Figure 1-3, so the machine is an 865 with two CPU's. Other cabinets in the
photo are peripheral controllers and, possibly, ESM or smaller systems such
as a Cyber 730.

**Reply by KC5VDJ** [reply]
I was thinking it would kill the latency too, but, maybe the IOUs?  It is described as an 875, the cabinet doors are close to the pic in the manual, and not roughly equal sized like the 865.  I was thinking it could have been an 865 misdescribed, but the doors are 875 per the manual pic.

As far as separating memory, ever read the NASA report on the ETA-10?  Thus the question mark in the subject.

**Comment by nickglazzard1959** [comment]
Hi Kevin,
I'm pretty sure the machine on the right is a 6500 not an 865. My reason
for saying that is this photograph:
[image: [7812508X.jpg]](https://drive.google.com/file/d/1bF1NgmN4dgrMYVew3XNRzUME8gH3t9kJ/view?usp=sharing)
Zooming into that, the T-shaped CPU boxes have "6500" logos on them. I
believe this photo was taken at the same time as the original photo in this
discussion, and as evidence of that, the chap in the blue boiler suit is
present in both (standing here and sitting in the original photo).
I'm pretty sure this was taken in 1978 or 1979 and comes from an era before
the 865 and 875.
But I might be wrong ... and I would really like to be, because I would
really like to have a high quality image of a Cyber 865 or 875 ... :-) ...
All the best,
Nick.

**Reply by KC5VDJ** [reply]
I'm seeing it now, definitely a 6500.  Gawd, Cray's toy sure was a pretty machine, that 7600...

**Comment by kej715** [comment]
Aha! I just assumed that the photo was taken during the 865/875 era. If it
was taken in 1978, then you're absolutely correct that the machine at the
right couldn't possibly be an 865, and it makes sense that it could be a
6500 where one bay of the "T" configuration is memory and IOU, and the
other two bays are CPU's.

**Reply by KC5VDJ** [reply]
I'm thinking that someone screwed up the pic for the cern page now, easy mistake.  Those do resemble the 170-865 though), that and I wasn't expecting such an error from cern 😉

**Comment by nickglazzard1959** [comment]
(This thing seems to want to show bold text here. Don't know why, it's not my intention to shout :-) )
It's a bit worrying for historical accuracy that CERN seem to have misfiled this photo. There is a comment in the discussion section of the CERN Document Server entry for this that reads:
---
[Jean de La Taille](https://cds.cern.ch/yourmessages/write?msg_to=Jean%20de%20La%20Taille&ln=en) 12 Jan 2019, 01:02 
This picture has been taken in 1975, absolutely not in 1983... In the picture, we can see the CDC 6400 (decomissioned before 1982) and the very old CDC 3400 (at the bottom, decomissioned before 1979).
---
and I think he's right (well, things say 6500 not 6400, but otherwise right).
The only reason I noticed this was that I got a copy of this photo from the CERN servers something like 15 years ago, and it was filed completely differently back then. It was all in simple date order and 1978 rings a bell. But it could be 1975. As it is now, anyone looking for 865/875 material will naturally think this it -- but it isn't. A little disappointing.

**Reply by KC5VDJ** [reply]
And the machines are similar enough visually to the 800 series to give it credibility.  Fooled me.  Add that in the pic published, you can't read the numbers , although you can suss out "control data" through the blur.  Also now I'm sussing the 6000-whatever through the blur knowing what it is.

**Comment by KC5VDJ** [comment]
I was thinking of closing the discussion, since this is in obvious error, but I agree, someone needs to tell cern...  let's all do it, maybe it make it to someone.

**Comment by nickglazzard1959** [comment]
Well, more by luck than judgement, I think I have found pictures of the 875 and 835 in the CERN data centre. I wish there were more at higher resolution, but these are the only ones I've located. There may even be the condensing unit(s) I was looking for in there too. Certainly a few 885 drives, etc. The 7600 is still there at this point too.
![8311749X](https://github.com/kej715/DtCyber/assets/27016608/8ed7ec05-872b-4583-9751-8c3a5d6d39c3)
<img width="807" alt="Screenshot 2023-09-07 at 17 30 51" src="https://github.com/kej715/DtCyber/assets/27016608/83d870ba-3d81-483e-8a23-a05a387afe01">

**Reply by KC5VDJ** [reply]
Looks bigger than I thought.  Too bad about the uptime issues ruining it for CDC at CERN.  I just read the history linked elsewhere in this thread.  I get the ground loop thing, those can be difficult beasts to deal with, but there had to be other issues for that lousy of uptime.

What is the round thing on the end of the far right cabinet in the b&w pic?  A recording hygrometer?  In a room that big, I'd hope a few were scattered around.  Humidity could be a b...., Hmmm phenolic or fiberglass/epoxy pcbs (I don't actually recall, I usually walked past engineers on the job)?  Which type makes a difference.

The 7600 vs everything else...  Always loved Cray's sense of aesthetics.  When you pay for best in class, it should look best in class.  A good philosophy (backfired on A.H.Grebe though when his pistol didn't in 1931, he made beautiful devices too, and refused to lower his standards at a really bad time).

**Comment by nickglazzard1959** [comment]
Here is the relevant URL: https://cds.cern.ch/record/970274 (forgot!)

**Comment by nickglazzard1959** [comment]
This is a very good summary of CERN's computer systems up to 1990, BTW ... Came across this quite a while ago, but it makes interesting reading. One line near the end may be changing now though: "Contrary to popular scenarios in the 50's, and in spite of their shortcomings, human beings are still in the driver's seat." For how much longer, though? ... :-) ...

**Comment by nickglazzard1959** [comment]
https://cdsweb.cern.ch/record/205360/files/p1.pdf

**Reply by KC5VDJ** [reply]
I kinda chuckled at the end, parallelism and vectorizing is the perennial problem, and probably always will be in an n-core/CPU n-CPU/System world, with ubiquitous GPUs using opencl / cuda / whatever, right down to the smallest devices... Fortran and Ada now standardizing the probable transparent means for using GPUs (parallel do, etc)....the assumed soon to happen increased complexity of compilers due to making the GPU transparent in Fortran and Ada....

The cyber uptime issue was unacceptable though, anyone would probably agree on that...same crap on a larger scale killed ETA.  One would have figured BE to be far more stable, but he was talking ground loops and hardware issues local to CERN...

I'd still like to see the cyber MAPs in DtCyber, using Intel HPF on Intel, and gfortran elsewhere (until more competent options arise on ARM & RISCV (the latter, I'd like to see the concept of GUARDIAN NonStop in as well...  Fault-tolerant space/nuke rated is now silicon, a perfect application for the concepts)).

He did get the Fortran thing right though, still standing tall....

**Comment by nickglazzard1959** [comment]
Good question! And that's an interesting possibility. Something in there
must be a 10 ton condensing unit, too and 2 x 5 ton cooling units. Which I
would like to identify, but ...
